mod-orders-storage/schemas/piece.json

```diff
@@ -49,6 +49,10 @@
       "description": "Receiving tenant for ECS-enabled clusters",
       "type": "string"
     },
+    "bindItemTenantId": {
+      "description": "Bind item tenant for ECS-enabled clusters",
+      "type": "string"
+    },
```

Comment:
lets move near bindItemId

Reply:
I think it is our last schema change in this release. Probably let's update all acq modules with schema change update

Reply:
Moved under bindItemId

Reply:
Will update all modules once this is merged then